src/sentry/static/sentry/app/views/organizationGroupDetails/header.tsx

```diff
@@ -219,54 +219,61 @@ class GroupHeader extends React.Component<Props, State> {
         />
         <GroupActions group={group} project={project} disabled={isReprocessing} />
         <NavTabs>
-          <ListLink
+          <StyledListLink
             to={`${baseUrl}${location.search}`}
             isActive={() => currentTab === TAB.DETAILS}
           >
             {t('Details')}
-          </ListLink>
-          <ListLink
+          </StyledListLink>
+          <StyledListLink
             to={`${baseUrl}activity/${location.search}`}
             isActive={() => currentTab === TAB.COMMENTS}
           >
             {t('Activity')} <Badge text={group.numComments} />
-          </ListLink>
-          <ListLink
+          </StyledListLink>
+          <StyledListLink
             to={`${baseUrl}feedback/${location.search}`}
             isActive={() => currentTab === TAB.USER_FEEDBACK}
           >
             {t('User Feedback')} <Badge text={group.userReportCount} />
-          </ListLink>
+          </StyledListLink>
           {hasEventAttachments && (
-            <ListLink
+            <StyledListLink
               to={`${baseUrl}attachments/${location.search}`}
               isActive={() => currentTab === TAB.ATTACHMENTS}
             >
               {t('Attachments')}
-            </ListLink>
+            </StyledListLink>
           )}
-          <ListLink
-            to={`${baseUrl}tags/${location.search}`}
-            isActive={() => currentTab === TAB.TAGS}
-          >
-            {t('Tags')}
-          </ListLink>
-          <ListLink to={eventRouteToObject} isActive={() => currentTab === 'events'}>
-            {t('Events')}
-          </ListLink>
-          <ListLink
-            to={`${baseUrl}merged/${location.search}`}
-            isActive={() => currentTab === TAB.MERGED}
-          >
-            {t('Merged Issues')}
-          </ListLink>
-          {hasSimilarView && (
-            <ListLink
-              to={`${baseUrl}similar/${location.search}`}
-              isActive={() => currentTab === TAB.SIMILAR_ISSUES}
-            >
-              {t('Similar Issues')}
-            </ListLink>
+          {!isReprocessing && (
+            <React.Fragment>
+              <StyledListLink
+                to={`${baseUrl}tags/${location.search}`}
+                isActive={() => currentTab === TAB.TAGS}
+              >
+                {t('Tags')}
```

Comment:
We've been trying to take a "disable don't hide" approach with the UI could these tabs be disabled instead of not visible?